app/controllers/result_filters/provider_controller.rb

```diff
@@ -0,0 +1,29 @@
+module ResultFilters
+  class ProviderController < ApplicationController
+    include FilterParameters
+
+    def new
+      if params[:query].blank?
+        flash[:error] = "Training provider"
+        return redirect_back
+      end
+
+      @provider_suggestions = Provider
+        .select(:provider_code, :provider_name)
+        .where(recruitment_cycle_year: "2020")
```

Comment:
Not liking this being hard-coded here. We should at least use `Settings.current_cycle`. There may also be an argument for letting the API decide what the current cycle is, which would mean not using `where(recruitment_cycle_year: ...)` at all here, but that should be a decision taken across the app.

Reply:
Also, this sounds like a candidate for a "scope", or since JsonApiClient doesn't support those directly, could it be a method: `Provider.in_current_cycle` (or similar)?